package.json

```diff
@@ -23,6 +23,7 @@
     "object-describer": "https://github.com/kubernetes-ui/object-describer.git#v1.0.4",
     "patternfly": "3.35.1",
     "patternfly-bootstrap-combobox": "1.1.7",
+    "patternfly-react": "1.19.1",
```

Comment:
New dependency; but for the record, I'm okay with this, we want to try and replace some cockpit specific components with pf-react anyway.